CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,37 @@
 # Code of Conduct
 
-The [Node.js Code of Conduct][] applies to this team.
+The OpenJS Foundation and its member projects use the Contributor
+Covenant v1.4.1 as its Code of Conduct. Refer to the following
+for the full text:
+
+* [english](https://www.contributor-covenant.org/version/1/4/code-of-conduct)
+* [translations](https://www.contributor-covenant.org/translations)
+
+Refer to the section on reporting and escalation in this document for the specific emails that can be used to report and escalate issues.
+
+# Reporting
+
+## Project Spaces
+
+For reporting issues in spaces related to a member project please use the email provided by the project for reporting. Projects handle CoC issues related to the spaces that they maintain.  Projects maintainers commit to:
+
+* maintain the confidentiality with regard to the reporter of an incident
+* to participate in the path for escalation as outlined in
+  the section on Escalation when required.
+
+## Foundation Spaces
+For reporting issues in spaces managed by the OpenJS Foundation, for example, repositories within the OpenJS organization, use the email `report@openjsf.org`. The CrossProjectCouncil (CPC) is responsible for managing these reports and commits to:
+
+* maintain the confidentiality with regard to the reporter of an incident
+* to participate in the path for escalation as outlined in
+  the section on Escalation when required.
+
+# Escalation
+
+The OpenJS Foundation maintains a Code of Conduct Panel (CoCP). This is a cross-foundation team established to manage escalation when a reporter believes that a report to a member project or the CPC has not been properly handled. In order to escalate to the CoCP send an email to `coc-escalation@openjsf.org`.
```

Comment:
`cross-foundation`? That suggests spanning multiple foundations (to me, at least). You mean `cross-project` (spanning multiple projects?) and/or `foundation-spanning` (spanning the entire foundation)?

Reply:
@Trott, how about foundation-wide ?

Reply:
> @Trott, how about foundation-wide ?

Yes, that's better than my suggested `foundation-spanning`. 👍